chain/bitcoind_client.go

```diff
@@ -364,6 +365,21 @@ func (c *BitcoindClient) Start() error {
 		return errors.New("mismatched networks")
 	}
 
+	// Get initial conditions.
```

Comment:
Why do this outside of establishing a connection? As is now, it's possible that we have stale information by the time the client is actually connected.

Reply:
I did it this way rather than within the goroutine (as it was done before) so that we actually return the error to the caller, signifying that something could potentially be wrong. FWIW, it will be updated within `ntfnHandler` as soon as a new block notification arrives from `bitcoind`.